pages/index.js

```diff
@@ -67,6 +87,22 @@ export default function HomePage() {
 
   return (
     <StyledDiv>
+      <button /* Only for testing of success-marker for past days,
+        it creates the saved daily entry for yesterday.
+        Clicking with default local storage values will mark yesterday either
+        red or green depending on the currently displayed calorie count.
+        The history with entries is also logged in the console.*/
+        onClick={() => {
+          const today = new Date();
+          const day = today.getDate() - 1;
+          const month = today.getMonth();
+          const year = today.getFullYear();
+          addHistoryEntry(year, month, day, isGoalExceeded);
+          console.log("history: ", history);
```

Comment:
```suggestion
```

Reply:
Why wouldn't I log the state, which all of my calendar functions will depend of + I can see of all the data is stored correctly?

Reply:
That's true for development, but you (and your team) don't want to have console.logs in the main branch. This is why we delete them before merging.